src/xcode/ENA/ENA/Source/Services/HealthCertificateValidationService/HealthCertificateValidationService.swift

```diff
@@ -83,52 +111,60 @@ final class HealthCertificateValidationService: HealthCertificateValidationProvi
+	
+	// MARK: - Onboarded Countries
+	
 	private func processOnboardedCountriesResponse(
 		packageDownloadResponse: PackageDownloadResponse,
 		completion: @escaping (Result<[Country], ValidationOnboardedCountriesError>) -> Void
 	) {
-		Log.info("Successfully received onboarded countries package. Proceed with eTag verification...")
+		Log.info("Successfully received onboarded countries package. Proceed with eTag verification...", log: .vaccination)
 
 		guard let eTag = packageDownloadResponse.etag else {
-			Log.error("ETag of package is missing. Return with failure.")
+			Log.error("ETag of package is missing. Return with failure.", log: .vaccination)
 			completion(.failure(.ONBOARDED_COUNTRIES_JSON_ARCHIVE_SIGNATURE_INVALID))
 			return
 		}
 		
-		Log.info("Successfully verified eTag. Proceed with package extraction...")
+		Log.info("Successfully verified eTag. Proceed with package extraction...", log: .vaccination)
 				
 		guard !packageDownloadResponse.isEmpty,
 			  let sapDownloadedPackage = packageDownloadResponse.package else {
-			Log.error("PackageDownloadResponse is empty. Return with failure.")
+			Log.error("PackageDownloadResponse is empty. Return with failure.", log: .vaccination)
 			completion(.failure(.ONBOARDED_COUNTRIES_JSON_ARCHIVE_FILE_MISSING))
 			return
 		}
-		Log.info("Successfully extracted sapDownloadedPackage. Proceed with package verification...")
+		Log.info("Successfully extracted sapDownloadedPackage. Proceed with package verification...", log: .vaccination)
 		
 		guard self.signatureVerifier.verify(sapDownloadedPackage) else {
-			Log.error("Verification of sapDownloadedPackage failed. Return with failure")
+			Log.error("Verification of sapDownloadedPackage failed. Return with failure", log: .vaccination)
 			completion(.failure(.ONBOARDED_COUNTRIES_JSON_ARCHIVE_SIGNATURE_INVALID))
 			return
 		}
-		Log.info("Successfully verified sapDownloadedPackage. Proceed now with CBOR decoding...")
+		Log.info("Successfully verified sapDownloadedPackage. Proceed now with CBOR decoding...", log: .vaccination)
 		
-		self.countryCodes(sapDownloadedPackage.bin, completion: { result in
-			switch result {
-			case let .success(countries):
-				Log.info("Successfully decoded country codes. Returning now.")
-				// Save in success case for caching
-				let receivedOnboardedCountries = ValidationOnboardedCountriesCache(
-					onboardedCountries: countries,
-					lastOnboardedCountriesETag: eTag)
-				store.validationOnboardedCountriesCache = receivedOnboardedCountries
-				completion(.success(countries))
-			case let .failure(error):
-				Log.error("Could not decode CBOR from package with error:", error: error)
-				completion(.failure(error))
+		self.countryCodes(
+			cborData: sapDownloadedPackage.bin,
+			completion: { result in
+				switch result {
+				case let .success(countries):
+					Log.info("Successfully decoded country codes: \(private: countries). Returning now.", log: .vaccination)
```

Comment:
Do the countries need to be private?

Reply:
Hm good question. But I thought its better to be safer then be sloppy for the privacy.